client/app.jsx

```diff
@@ -22,33 +23,30 @@ export default function App() {
   useEffect(() => {
     window.addEventListener('hashchange', handleChange);
     return () => window.removeEventListener('hashchange', handleChange);
-
   }, []);
 
   function handleClick() {
     setIsVisible(!isVisible);
   }
 
-  function handleAnchorClick() {
-    setIsClicked(!isClicked);
+  function onAnchorClick() {
```

Comment:
Actually, `handleAnchorClick` would fit better into React's suggested naming style. The rule they suggest is:

* If it _handles_ an event, name it `handleXxx`. Event handlers are functions that do something when an event happens.
* If it _declares_ an event, name it `onXxx`. You declare events by accepting them as `props` in a component, and the component calls the function when the event happens. So `onAnchorClick` would be a prop in the App component that is called when the user clicks on the anchor.